### PR TITLE
Add a local postgres server

### PR DESCRIPTION
**Pull request by MichaelDeng03**
Adds a dockerized local postgres server. Local frontend uses this database once you add `SQLALCHEMY_DATABSE_URL="postgresql+psycopg://local_user:local_password@localhost:5432/local_db"` to `.env`, and run `make run-postgres`.

I created a postgres instance in EC2 as well, but until FE is deployed/other services need the instance, I can't really test it. ATM it's just floating in ec2 land doing nothing.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -122,11 +130,3 @@ local-webapp-main:
...
 	/bin/bash -c "set -a; source .env.daemons; set +a; cd webapp/daemons && $(POETRY) fastapi dev server.py"
 
-
-# ------------------------ Tailscale -----------------------
-build-tailscale:
-	docker compose -f docker/docker-compose.yml build tailscale
-
-run-tailscale:
-	docker compose -f docker/docker-compose.yml down tailscale 
-	docker compose -f docker/docker-compose.yml up -d tailscale
```
Comment (kgang) [inline Makefile:132]: how come these are rm'd?

**Review by kgang — APPROVED** [review]
neat

File: Makefile (hunk 1 above)
Comment (MichaelDeng03) [reply Makefile:132]: They shouldn't have made it in my tailscale pr [](edited-with-Axolo)

Comment (kgang) [reply Makefile:132]: sg [](edited-with-Axolo)

**Pull request merged**